docs/manager/graphql-reference/schema.graphql

```diff
@@ -598,6 +598,9 @@ type ImageNode implements Node {
   Added in 25.3.0. One of ['read_attribute', 'update_attribute', 'create_container', 'forget_image'].
   """
   permissions: [ImagePermissionValueField]
+
+  """Added in 25.11.0. Indicates if the image is installed on any Agent."""
+  installed: Boolean
```

Comment:
How about returning a list of AgentNode objects instead of exposing a boolean field like installed?
This would be useful on the frontend, not only to check whether the image is installed on one or more agents, but also to know exactly which agents the image is installed on.

Reply:
If we have to add `installed` in its current form, please add `installed` to the `queryfilter` as well.

Reply:
The client part requires only "installed" field. If we want to add a field of a list of agents, then we should resolve it as another Agent GQL query or Agent Relay node.
I want to keep the work small and we should discuss how we should add the field.
And the "installed" field is not able to filter. So I won't add it into query filter